src/services/twilio-help/twilio-command-help.js

```diff
@@ -31,6 +33,30 @@ class TwilioCommandHelp extends CommandHelp.default {
     return returnList.join('\n');
   }
 
+  // overriding to include help document link
+  examples(examples) {
```

Comment:
Any specific reason to override `examples` and not add additional func `docs()` instead? Do not see them linked, and this'd save over-writing the oclif code at our end (So it's better maintained by oclif team itself). Also, I see UTs not being covered for `examples` around lines #41-45 due to this and devoids us of catching any future bugs. 

PS: Could create additional func and add to the render list here: https://github.com/oclif/plugin-help/blob/master/src/command.ts#L40-L47

Reply:
Help docs looked relevant to examples, that's why but seeing the downsides of it, it'd be better to make our own func for it. Will add it, thanks. :)

Reply:
Modified this, [here](https://github.com/twilio/twilio-cli/blob/c42df8ba8850be5f826d6f8950fe164af19a3089/src/services/twilio-help/twilio-command-help.js#L37)